Tests/UserServiceTests/Utils/EditUserTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using DTO;
+using DTO.BrokerRequests;
+using DTO.RestRequests;
+using FluentValidation;
+using FluentValidation.Results;
+using MassTransit;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using UserService.Commands;
+using UserService.Interfaces;
+
+namespace UserServiceTests
+{
+    class EditUserTests
+    {
+        private Mock<IValidator<UserInfoRequest>> userValidatorMock;
+        private Mock<IValidator<PasswordChangeRequest>> passwordValidatorMock;
+        private Mock<IRequestClient<InternalEditUserInfoRequest>> requestClientMock;
+        private Mock<Response<OperationResult>> resultMock;
+        private IEditUserCommand command;
+
+        [SetUp]
+        public void Init()
+        {
+            OperationResult operation = new OperationResult();
+            operation.IsSuccess = true;
+            resultMock = new Mock<Response<OperationResult>>();
+            resultMock
+                .Setup(x => x.Message)
+                .Returns(operation);
+            requestClientMock = new Mock<IRequestClient<InternalEditUserInfoRequest>>();
```

Comment:
Spaces between different initializations will be very useful for easier reading

Reply:
fixed